### PR TITLE
Improve integrations section examples

### LINKED ISSUE #5857: Improve Slack, PagerDuty and Shuffle integrations documentation
While testing [Release 4.4.0 - Alpha 2 - E2E UX tests - Slack + Pagerduty + Shuffle](https://github.com/wazuh/wazuh/issues/15766) where integrations are required I have found the documentation to integrate that software a bit incomplete.

There is documentation about how to modify the `ossec.conf` but not how to perform the setup and configuration of the external software. It could be explained, for example, in more detail how to obtain the API keys.

In the testing issue, I have tried to explain the steps to correctly deploy the external software integrations.

### PR DESCRIPTION
**Pull request by javimed**
## Description
This PR addresses and closes #5857 . It expands on the integrations examples.
 
## Checks
- [x] Compiles without warnings.
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

**Review by mjcr99 — APPROVED** [review]

File: source/user-manual/manager/manual-integration.rst [hunk 1]
```diff
@@ -55,89 +55,142 @@ The full configuration reference for the Integrator daemon can be found :ref:`he
 Slack
 -----
 
-This integration allows receiving alerts into a Slack channel thanks to the `Incoming Webhooks <https://api.slack.com/incoming-webhooks>`_, a simple way to post messages from 3rd-party apps (in this case, Wazuh).
+This integration uses Slack Incoming Webhooks and allows posting Wazuh alerts into a Slack channel.
 
-This is an example configuration for the Slack integration:
+To set up this integration follow these steps.
```
Comment (Arcapas) [inline source/user-manual/manager/manual-integration.rst:60]:
```suggestion
To set up this integration, follow these steps.
```

File: source/user-manual/manager/manual-integration.rst [hunk 2]
```diff
@@ -55,89 +55,142 @@ The full configuration reference for the Integrator daemon can be found :ref:`he
...
+  :width: 50%
 
 PagerDuty
 ---------
 
-`PagerDuty <https://www.pagerduty.com/>`_ is a SaaS incident response platform suitable for IT departments. This integration allows creating a service using its official API in order to receive Wazuh alerts on the Incidents Dashboard.
+`PagerDuty <https://www.pagerduty.com/>`_ is a SaaS incident response platform suitable for IT departments. The Pagerduty integration uses the Pagerduty API to forward Wazuh alerts to its Incidents Dashboard.
 
-This is an example configuration for the PagerDuty integration:
+To set up this integration do the following.
```
Comment (Arcapas) [inline source/user-manual/manager/manual-integration.rst:90]:
```suggestion
To set up this integration, do the following.
```

File: source/user-manual/manager/manual-integration.rst [hunk 3]
```diff
@@ -55,89 +55,142 @@ The full configuration reference for the Integrator daemon can be found :ref:`he
...
+   .. include:: /_templates/common/restart_manager.rst
 
 Shuffle
 -------
 
-`Shuffle <https://shuffler.io/>`_ is an Open Source interpretation of SOAR. It aims to bring all the capabilities necessary to transfer data throughout an enterprise with plug-and-play Apps. This integration allows sending alerts into a Shuffle workflow thanks to the `Wazuh Webhook <https://shuffler.io/docs/extensions#wazuh>`_, a simple way to send alerts from Wazuh to Shuffle.
+`Shuffle <https://shuffler.io/>`__ is an Open Source interpretation of SOAR. It transfers data throughout the enterprise with plug-and-play Apps. The Shuffle integration allows forwarding Wazuh alerts into a Shuffle Workflow using a `webhook <https://shuffler.io/docs/triggers#webhook>`__.
 
-This is an example configuration for the Shuffle integration:
+To set up this integration do the following.
```
Comment (Arcapas) [inline source/user-manual/manager/manual-integration.rst:146]:
```suggestion
To set up this integration, do the following.
```

File: source/user-manual/manager/manual-integration.rst [hunk 4]
```diff
@@ -55,89 +55,142 @@ The full configuration reference for the Integrator daemon can be found :ref:`he
...
+        <name>pagerduty</name>
+        <api_key>API_KEY</api_key> <!-- Replace with your PagerDuty API key -->
+        <level>10</level>
+      </integration>
+
+#. Restart the Wazuh manager to apply the changes.
+
+   .. include:: /_templates/common/restart_manager.rst
+
+Once the configuration is complete, alerts start showing into the Pagerduty dashboard.
```
Comment (Arcapas) [inline source/user-manual/manager/manual-integration.rst:109]:
```suggestion
Once the configuration is complete, alerts start showing on the Pagerduty dashboard.
```

File: source/user-manual/manager/manual-integration.rst [hunk 5]
```diff
@@ -55,89 +55,142 @@ The full configuration reference for the Integrator daemon can be found :ref:`he
...
+   :align: center
+   :width: 80%
 
 VirusTotal
 ----------
 
 This integration allows the inspection of malicious files using the VirusTotal database. Find more information about this at the :ref:`VirusTotal integration <virustotal-scan>` page.
 
-This is an example configuration for the VirusTotal integration:
+To set up this integration follow these steps.
```
Comment (Arcapas) [inline source/user-manual/manager/manual-integration.rst:121]:
```suggestion
To set up this integration, follow these steps.
```

File: source/user-manual/manager/manual-integration.rst [hunk 6]
```diff
@@ -55,89 +55,142 @@ The full configuration reference for the Integrator daemon can be found :ref:`he
...
+
+.. thumbnail:: /images/manual/integration/pagerduty.png
+   :title: PagerDuty Incidents Dashboard
+   :align: center
+   :width: 80%
 
 VirusTotal
 ----------
 
 This integration allows the inspection of malicious files using the VirusTotal database. Find more information about this at the :ref:`VirusTotal integration <virustotal-scan>` page.
```
Comment (Arcapas) [inline source/user-manual/manager/manual-integration.rst:119]:
```suggestion
This integration allows the inspection of malicious files using the VirusTotal database. Find more information about this on the :ref:`VirusTotal integration <virustotal-scan>` page.
```

File: source/user-manual/manager/manual-integration.rst [hunk 7]
```diff
@@ -55,89 +55,142 @@ The full configuration reference for the Integrator daemon can be found :ref:`he
...
-   </integration>
+#. Set **Recipients** and **Subject** in the email configuration. Put ``$exec`` in the **Body** to include the alert information.
 
-.. note::
-   The Shuffle default installation is done using `docker-compose`. If the Wazuh installation is done on the same network as Shuffle, it's not necessary to specify port ``3001`` in the ``<hook_url>`` field.
+#. Add a webhook to the Workflow.
+
+#. Start the webhook and copy the webhook URL.
+
+#. Edit ``/var/ossec/etc/ossec.conf`` in the Wazuh server and include a configuration block such as the following. Replace ``https://shuffler.io/api/v1/hooks/webhook_WEBHOOK_ID`` with the webhook URL. The rule level filter is optional and you can remove it or set another level value for the integration.
```
Comment (Arcapas) [inline source/user-manual/manager/manual-integration.rst:156]:
```suggestion
#. Edit ``/var/ossec/etc/ossec.conf`` in the Wazuh server and include a configuration block such as the following. Replace ``https://shuffler.io/api/v1/hooks/webhook_WEBHOOK_ID`` with the webhook URL. The rule level filter is optional. You can remove it or set another level value for the integration.
```

File: source/user-manual/manager/manual-integration.rst [hunk 8]
```diff
@@ -55,89 +55,142 @@ The full configuration reference for the Integrator daemon can be found :ref:`he
...
+         <hook_url>https://shuffler.io/api/v1/hooks/webhook_WEBHOOK_ID</hook_url> <!-- Replace with your Shuffle hook URL -->
+         <level>3</level>
+         <alert_format>json</alert_format>
+      </integration>
+
+#. Restart the Wazuh manager to apply the changes.
+
+   .. include:: /_templates/common/restart_manager.rst
+
+Once the configuration is complete, alerts start showing into the email inbox.
```
Comment (Arcapas) [inline source/user-manual/manager/manual-integration.rst:172]:
```suggestion
Once the configuration is complete, alerts start showing in the email inbox.
```

File: source/user-manual/manager/manual-integration.rst [hunk 9]
```diff
@@ -55,89 +55,142 @@ The full configuration reference for the Integrator daemon can be found :ref:`he
...
+   :width: 80%
 
 Custom integration
 ------------------
 
-The integrator tool is able to connect wazuh with other external software. Read the `How to integrate external software using Integrator <https://wazuh.com/blog/how-to-integrate-external-software-using-integrator//>`_ document for more information.
+The integrator tool is able to connect Wazuh with other external software. Read the `How to integrate external software using Integrator <https://wazuh.com/blog/how-to-integrate-external-software-using-integrator//>`_ document for more information.
 
-This is an example configuration for a custom integration:
+Below, you can find an example of a configuration block in the ``ossec.conf`` file for a custom integration.
```
Comment (Arcapas) [inline source/user-manual/manager/manual-integration.rst:184]:
```suggestion
Below, you can find an example of a configuration block in the ``ossec.conf`` file for custom integration.
```

**Review by Arcapas — COMMENTED** [review]
I made some suggestions.

**Review by Arcapas — APPROVED** [review]
LGTM

File: source/user-manual/manager/manual-integration.rst [hunk 10]
```diff
@@ -55,89 +55,142 @@ The full configuration reference for the Integrator daemon can be found :ref:`he
...
+     </integration>
+
+#. Restart the Wazuh manager to apply the changes.
+
+   .. include:: /_templates/common/restart_manager.rst
+
+Once the configuration is complete, alerts start showing in the selected channel.
+
+.. thumbnail:: /images/manual/integration/slack.png
+  :title: PagerDuty Incidents Dashboard
```
Comment (javimed) [inline source/user-manual/manager/manual-integration.rst:81]:
```suggestion
  :title: Alerts in Slack channel
```

**Pull request merged**